pkg/middleware/request_tracing.go

```diff
@@ -3,20 +3,34 @@ package middleware
 import (
 	"fmt"
 	"net/http"
+	"strings"
 
 	opentracing "github.com/opentracing/opentracing-go"
 	"github.com/opentracing/opentracing-go/ext"
 
 	"gopkg.in/macaron.v1"
 )
 
-func RequestTracing(handler string) macaron.Handler {
+const routeOperationNameKey = "route.operation.name"
+
+func ProvideRouteOperationName(handler string) macaron.Handler {
+	return func(res http.ResponseWriter, req *http.Request, c *macaron.Context) {
+		c.Data[routeOperationNameKey] = handler
```

Comment:
Perhaps it's better to pass it in `context.Context` to avoid more depends on macaron?

Reply:
Good point. Will fix.